app/models/bishop.rb

```diff
@@ -2,4 +2,13 @@ class Bishop < Piece
   def to_unicode
     black? ? "\u265D" : "\u2657"
   end
+
+  def valid_move?(destination_x, destination_y)
+    origin_x = x_position
+    origin_y = y_position
+    distance_x = (destination_x - origin_x).abs
+    distance_y = (destination_y - origin_y).abs
+
+    distance_x == distance_y
```

Comment:
You could save all these 5 lines into 1 by doing this (destination_x - x_position).abs == (destination_y - y_position).abs.